src/main/java/it/gov/pagopa/pu/workflow/wf/paymentsreporting/pagopa/config/OrganizationPaymentsReportingPagoPaFetchWfConfig.java

```diff
@@ -9,7 +9,7 @@
 import org.springframework.context.annotation.Configuration;
 
 @Configuration
-@ConfigurationProperties(prefix = "workflow.payments-reporting-pago-pa")
+@ConfigurationProperties(prefix = "workflow.organization-payments-reporting-pago-pa")
```

Comment:
I would prefer a prefix which will give more importance to the context
```suggestion
@ConfigurationProperties(prefix = "workflow.payments-reporting-pagopa-organization-fetch")
```